lib/app/routes/github_callback.rb

```diff
@@ -21,7 +21,7 @@ def self.registered(app)
 
           path = params[:splat].first
           if path.nil? || path.empty?
-            redirect root_path
+            redirect dashboard_path
```

Comment:
Let's just hard-code the "/dashboard" here, same as the others, and not define a "dashboard_path" method.